### DISCUSSION TITLE
RAC's Compatibility with React Hook Form

### DISCUSSION THREAD

**Discussion by thevipinmishra** [adobe/react-spectrum#5175] [Q&A]
So, I'm trying to use RAC's Forms/Input related components with React Hook Form, so far I've managed to use most of them using the <Controller /> component, but I don't know how to use it with Combobox component, because the event props are different from the standard event names.

For example, `onInputChange` and `onSelectionChange`.

Since React Hook Form is a popular library, I'm assuming that a lot of people are going to use it.

Can we document it somewhere? maybe a section dedicated for such Form libraries?

**Comment by LFDanLu** [comment]
We can certainly look to add documentation for it if there is enough demand, feel free to create an issue so people can thumbs it up. That being said, there are some known issues with React Hook Form/Formik and our Combobox that haven't yet been fully investigated yet: https://github.com/adobe/react-spectrum/issues/4621.
